frontend/src/components/Dashboard.js

```diff
@@ -1,7 +1,27 @@
 import React from 'react'
 import {
-    HStack, VStack, Box, Center, ListItem, UnorderedList
-  } from "@chakra-ui/react"
+    HStack, VStack, Box, Center, ListItem, UnorderedList} from "@chakra-ui/react"
+
+  const axios = require('axios');
+
+  function loadWeakWords() {
+    axios({
+        method: 'POST',
+        url: 'http://127.0.0.1:5000/api/weakwords',
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({
+            uid: localStorage.getItem('uid'),
+        }),
+    })
+    .then(function (response) {
+      if(response.status === 200){
+      }
+    })
+    .catch(function (error) {
+    });
+  }
+
+  
 
 const Dashboard = () => {
```

Comment:
Are you going to update the dashboard to use data from backend?

Reply:
yeah I created a PR updating the dashboard just now